app/controllers/application_controller.rb

```diff
@@ -5,6 +5,21 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "secret"
   end
 
-end
\ No newline at end of file
+  helpers do
+    def current_user
+      @user = User.find_by_id(session[:user_id])
```

Comment:
Yes, makes sense! Thanks! Is this what you were suggesting? 
`  def current_user
      @user ||= User.find_by_id(session[:user_id]) if session[:user_id]
    end`